src/GoogleCloudPubSub/GoogleCloudPubSub.ts

```diff
@@ -127,9 +127,12 @@ export class GoogleCloudPubSub implements GCPubSub {
 
     subscription.on('message', (message: Message): void => {
       const extendedMessage: ExtendedMessage<T> = new ExtendedMessage<T>(message);
+      const logMessage: ExtendedMessage<T> = new ExtendedMessage<T>(message);
 
+      // eslint-disable-next-line
+      delete (logMessage as any).originalMessage;
       this.logger.debug(
-        extendedMessage,
+        logMessage,
```

Comment:
Instead of instantiating a new extendedMessage, you can omit the property using omit from lodash
```suggestion
        omit(extendedMessage, "originalMessage")

```

Reply:
well, avoid lodash if you can do it natively 😄

Reply:
I agree, we should opt-in for the native choices rather than relying a 3rd party lib 😉 

For example, lodash has removed `omit` function due to its poor performance 😛 
https://levelup.gitconnected.com/omit-is-being-removed-in-lodash-5-c1db1de61eaf

That function is removed so I couldn't see the source code to know how it is implemented. If it is similar to the destructuring then I doubt it will work in this case because `originalMessage` is a private property and not easily referenced from outside.